src/include/uapi/ipc/header.h

```diff
@@ -239,6 +239,7 @@ struct sof_ipc_dsp_oops_plat_hdr {
 	uint32_t stackoffset;	/* Offset to stack pointer from beginning of
 				 * oops message
 				 */
+	uintptr_t stack_ptr;	/* Stack ptr */
```

Comment:
If this structure used in kernel?
Need a ABI change?

Reply:
No dependency on the kernel driver. The core dump is parsed by the tool being changed within the same commit.